### ISSUE TITLE
Toski can't find my MIT Scheme installation

### ISSUE THREAD

**Issue by mdwhatcott** [kennethshawfriedman/Toski#3]
Not sure why Toski can't see my installation. Is something supposed to be running?

![Screen Shot 2019-07-27 at 11 32 04 AM](https://user-images.githubusercontent.com/202910/61997674-515f6280-b062-11e9-8902-465adbbe8929.png)

![Screen Shot 2019-07-27 at 11 32 41 AM](https://user-images.githubusercontent.com/202910/61997676-56bcad00-b062-11e9-8b0b-6652c1364f72.png)

**Comment by kennethshawfriedman** [comment]
Ahh, sorry about that!

No, you shouldn't have to have anything running. As long as Scheme is on the machine, it should work.

I will do some digging and see if I can find out why that's happening for you!

(Quick question: if you simply run "mit-scheme" from the command line, does a Scheme instance start?)

**Comment by mdwhatcott** [comment]
Thanks for the quick reply. I'm totally new to scheme (reading SICP for the first time) so I'm probably missing something obvious to an experienced schemer.

Your question about being able to run `mit-scheme` from the command line got me thinking. Even when I added the app folder to my path it didn't work. Then I realized that scheme is installed to a `/Applications/MIT:GNU Scheme 10.1.9.app`, a folder that contains a colon `:` (the character used to separate $PATH entries). So, after renaming the folder to something more sensible and adding it to my $PATH, I was able to invoke it from the command line and the app started up and worked like a charm.

I'm guessing that the app depends on `mit-scheme` being accessible from the `$PATH`?

Anyway, thanks for building this! I'm glad to have a lightweight tool for tinkering.

![Screen Shot 2019-07-27 at 12 25 44 PM](https://user-images.githubusercontent.com/202910/61998197-ac488800-b069-11e9-81d8-3811f5c696c3.png)
